woudc_data_registry/models.py

```diff
@@ -299,17 +303,19 @@ class Instrument(base):
     # relationships
     station = relationship('Station', backref=__tablename__)
     dataset = relationship('Dataset', backref=__tablename__)
+    deployment = relationship('Deployment', backref=__tablename__)
 
     def __init__(self, dict_):
         self.station_id = dict_['station_id']
         self.dataset_id = dict_['dataset_id']
+        self.contributor_id = ':'.join([dict_['contributor'],
+                                        dict_['project']])
```

Comment:
You could even get away without this, by doing `self.deployment_id = ':'.join([station_id, contributor, project])` down below in `generate_ids` instead of using this temporary contributor ID.

`dict_` omitted left out because why not?